ambari-server/src/main/java/org/apache/ambari/server/utils/JsonUtils.java

```diff
@@ -39,6 +40,9 @@
   public static JsonParser jsonParser = new JsonParser();
 
   private static final ObjectMapper JSON_SERIALIZER = new ObjectMapper();
+  static {
+    JSON_SERIALIZER.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+  }
```

Comment:
`setSerializationInclusion` and other similar methods return the mapper, so it can be chained like:

```
private static final ObjectMapper JSON_SERIALIZER = new ObjectMapper()
  .setSerializationInclusion(JsonInclude.Include.NON_NULL);
```